### ISSUE TITLE
facing issue while using "ExcelDataReader"

### ISSUE THREAD

**Issue by MuslehAbulHaj** [ExcelDataReader/ExcelDataReader#619]
I got this message "System.Xml.XmlException: 'Root element is missing.'" while the code below:

DataTableCollection tableCollection;
        private void BtnOpenFile_Click(object sender, EventArgs e)
        {

            using (OpenFileDialog openFileDialog = new OpenFileDialog() { Filter = "Excel Files|*.xlsx" })
            {
                if (openFileDialog.ShowDialog() == DialogResult.OK)
                {
                    TBoxFilePath.Text = openFileDialog.FileName;
                    FileInfo fi = new FileInfo(openFileDialog.FileName);
                    string extn = fi.Extension;

                    using (var stream = File.Open(fi.FullName, FileMode.Open, FileAccess.Read))
                    {
                        //System.Text.Encoding.RegisterProvider(System.Text.CodePagesEncodingProvider.Instance);
                        using (var reader = ExcelReaderFactory.CreateReader(stream))
                        {
                            var result = reader.AsDataSet(new ExcelDataSetConfiguration()
                            {
                                ConfigureDataTable = (_) => new ExcelDataTableConfiguration()
                                {
                                    UseHeaderRow = true
                                }
                            });
                            

                            tableCollection = result.Tables;
                            SheetDdl.Items.Clear();
                            foreach (DataTable table in tableCollection)
                                SheetDdl.Items.Add(table.TableName);//add sheet to combobox
                        }

                    }
                }

            }
        }

**Comment by appel1** [comment]
Can you open the file in Excel? If Excel can open the file, could you provide the file for us to reproduce the issue?

**Comment by MuslehAbulHaj** [comment]
your reply was so useful ... i just realized that the Excel file has issues to open. thanks for your smart reply. 👍

But still have different error when I tried to open another Excel-File, which is:
"System.IO.IOException: 'The process cannot access the file 'G:\EFILE\DPA\Book1.xlsx' because it is being used by another process.'"
I tried find out why it's showing that error, by separate Read-file-path Process with a button & read Content with another Button... but still same that error ... knowing the ms-excel is closed already & no other processes on the same is running!
